homeassistant/components/homekit_controller/const.py

```diff
@@ -107,3 +107,10 @@
     EncryptionError,
     AccessoryDisconnectedError,
 )
+
+# 10 seconds was chosen because its soon enough
```

Comment:
it's

Reply:
https://github.com/home-assistant/core/pull/82111